debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/DebeziumContainerTest.java

```diff
@@ -48,17 +48,14 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DebeziumContainerTest.class);
 
-    private static final DockerImageName POSTGRES_DOCKER_IMAGE_NAME = DockerImageName.parse("debezium/postgres:11")
-            .asCompatibleSubstituteFor("postgres");
-
     private static final Network network = Network.newNetwork();
 
+    private static final PostgresInfrastructure POSTGRES_INFRASTRUCTURE = PostgresInfrastructure.getDebeziumPostgresDefaultInfrastructure(network);
+
     private static final KafkaContainer kafkaContainer = new KafkaContainer()
             .withNetwork(network);
 
-    public static PostgreSQLContainer<?> postgresContainer = new PostgreSQLContainer<>(POSTGRES_DOCKER_IMAGE_NAME)
-            .withNetwork(network)
-            .withNetworkAliases("postgres");
+    public static PostgreSQLContainer<?> postgresContainer = POSTGRES_INFRASTRUCTURE.getPostgresContainer();
 
     public static DebeziumContainer debeziumContainer = new DebeziumContainer("debezium/connect:1.1.1.Final")
```

Comment:
@jpechane I have added one for `Apicurio`. How you want to add for `Connect` image? I guess we have simply passed the image name into `DebeziumContainer`.